packages/components/script.tsx

```diff
@@ -64,6 +65,11 @@ const Script: React.FC<ScriptProps> = ({ src, code, id }) => {
       // Add the ID if specified.
       if (id) script.id = id;
 
+      // Add any other props to the internal <script> tag.
+      for (let key of Object.keys(props)) {
```

Comment:
Here you can use `for ... in` to iterate over all the keys directly.

```suggestion
      for (const key in props) {
```